change-notes/1.23/analysis-javascript.md

```diff
@@ -38,3 +38,4 @@
 ## Changes to QL libraries
 
 * `Expr.getDocumentation()` now handles chain assignments.
+* Added `Deferred` as a promise library in Promises.qll
```

Comment:
Improvements to library models aren't usually listed in this section. This is mainly for API changes that are relevant for QL writers. There is no need to use the `Deferred` module directly, and certainly no reason to import `Promises.qll` directly.

You could add a bullet in the general improvements section, saying that promises derived from a `Deferred` object are now recognized.